package/libs/libnl/patches/101-add-musl-workaround-to-the-libc-compat.h-copy.patch

```diff
@@ -1,51 +1,35 @@
 --- a/include/linux-private/linux/if_ether.h
```

Comment:
This patch should not be needed any more, this upstream commit should fix the problem with musl libc:
https://github.com/thom311/libnl/commit/73c1d0479643cf743b72879084e70f738f11bded